javascript/1898-maximum-number-of-removable-characters.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * https://leetcode.com/problems/maximum-number-of-removable-characters/
+ * 
+ * Brute force
+ * Time O(removable.length * s.length) | Space O(1)
+ * @param {string} s
+ * @param {string} p
+ * @param {number[]} removable
+ * @return {number}
+ */
+var maximumRemovals1 = function(s, p, removable) {
+  let k = 0;
+  // removable.reverse();
+  s = s.split('');
+  p = p.split('');
+
+  for (let i = 0; i < removable.length; i++) {
+    s[removable[i]] = -1;
+    if (isSubSet1(s, p)) {
+      k++;
+    } else {
```

Comment:
Avoid else

Reply:
Done.

Reply:
@aakhtar3, It's done. I  removed the else case.